sequence_Helicopter/modules/gc_and_quality_calculator.py

```diff
@@ -0,0 +1,33 @@
+# GC-состав
+
+def calculate_gc_content(sequence: str) -> float:
+    """
+    Calculates the GC content percentage in a sequence.
+
+    Arguments:
+    sequence -- a string representing a DNA or RNA sequence.
+
+    Returns:
+    The percentage of GC content in the sequence (float).
+    """
+    g_count = sequence.count("C")
+    c_count = sequence.count("G")
+    gc_content = (g_count + c_count) / len(sequence) * 100
+    return gc_content

```

Comment:
По идее можно в одну строчку прописать.  Точно можно присваивание убрать. Этот комментарий в целом не только тут актуален.
```suggestion
    g_count = sequence.count("C")
    c_count = sequence.count("G")
    return (g_count + c_count) / len(sequence) * 100
```